src/instrumentsscene/qml/MuseScore/InstrumentsScene/InstrumentsPanel.qml

```diff
@@ -106,6 +106,9 @@ Item {
 
             onRemovingRequested: {
                 instrumentsTreeModel.removeSelectedRows()
+                while (instrumentsTreeModel.isEmpty) {
```

Comment:
I don't think this is the right way to solve the problem. At the very least, this solution should be coordinated with the  design team (@Tantacrul or @bkunda)

Reply:
In principle, I think it's okay to immediately prompt the user to add instruments if they, for whatever reason, decide to delete every instrument in their score (via the Instruments panel). What I don't like is the persistently re-appearing "Add instruments" dialog whenever the "Cancel" button is clicked. Perhaps better would be to trigger a modal dialog whenever "Cancel" is clicked _in this specific context_, which reads something along the lines of "Please add at least one instrument to continue working in this score" (And a nice-to-have might then be a "Restore deleted instruments" button so the user can return to the stage they were at before this dialog was automatically re-triggered).  

In saying this, the solution presented here does leave open a loophole, which is where the user has more than one project open at once. Quitting MuseScore from another opened project allows you to save the "empty" one without having to add any new instruments. It is then possible to open the "empty" score, in which any interaction with the playback panel continues to trigger a crash. Fixing _this_ issue would then give rise to some further UX complications, like forcing the user to add some instruments before being able to save and close their session (which is becoming too messy from my POV). 

In light of this, if there is another technical solution that doesn't change any of the present UX, I'd say let's explore that first.